x-pack/plugins/reporting/server/lib/enqueue_job.ts

```diff
@@ -47,15 +47,13 @@ export function enqueueJobFactory(
       reporting.getStore(),
     ]);
 
-    const config = reporting.getConfig();
     const job = await createJob!(jobParams, context, request);
 
     // 1. Add the report to ReportingStore to show as pending
     const report = await store.addReport(
       new Report({
         jobtype: exportType.jobType,
         created_by: user ? user.username : false,
-        max_attempts: config.get('capture', 'maxAttempts'), // NOTE: since max attempts is stored in the document, changing the capture.maxAttempts setting does not affect existing pending reports
```

Comment:
Max attempts is set on the report once it is claimed. The main reason to do this is to show the information in the "info" panel in the UI.